.github/workflows/deploy-auto.yml

```diff
@@ -0,0 +1,25 @@
+# Automatically create a new version/tag, if new commits are available since the last tag
+name: Auto-tagger
+
+on:
+  push:
+    tags:
+      - deploy
```

Comment:
I'm thinking this could be avoided altogether; if the tag isn't deleted in time by GHA, someone could fork it and keep the tag forever, which could cause trouble.

Reply:
Yeah, dropping it, it's not a big deal to manually creating the rare hotfix `20.5.10.1` tag